src/app/common/icon.js

```diff
@@ -38,7 +38,7 @@
     };
 
     function iconUrl (name) {
-      return $window.location.href + '#icon-' + name;
+      return $window.location.origin + $window.location.pathname + '#icon-' + name;
```

Comment:
@rupl That's the thing you were talking about in the standup about SVG icons correct ?

Reply:
yup, using `window.local.href` and using keyboard nav to "Skip to content" meant that SVGs were pointing at something like this: http://app.hid.vm/landing#main#icon-arrow with two anchors